src/tables.js

```diff
@@ -9,6 +9,7 @@ const Flow = {
   STEM_HEIGHT: 32,
   STAVE_LINE_THICKNESS: 2,
   RESOLUTION: 16384,
+  INITIAL_ARTICULATION_OFFSET: -0.5,
```

Comment:
I'm trying to minimize (or get rid of) these global constants. Prefer `class.CONSTANT` instead, so you could have `Articulation.INITIAL_OFFSET` located in `articulation.js`.